### PR TITLE
Haolin/evaluation

### PR DESCRIPTION
**Pull request by hlnchen**

### CODE REVIEW

File: torchprime/torch_xla_models/flex/modeling_qwen.py [hunk 1]
```diff
@@ -422,9 +418,10 @@ def forward(
   ) -> tuple[torch.FloatTensor, torch.FloatTensor | None]:
     if not self.training:
       # haolin: during inference the masking is done when preprocessing the input, we don't need src_mask and noising
-      hidden_states = self.model(input_ids=input_ids, attention_mask=attention_mask)
-      logits = self.lm_head(hidden_states)
-      logits = logits.float()[..., :-1, :].contiguous() # NOTE: we don't need the logits of the last token
+      model_output = self.model(input_ids=input_ids, attention_mask=attention_mask)   
+      hidden_states = model_output
+      logits = self.lm_head(hidden_states) # NOTE: we shift logits in generate()
+      # logits = logits.float()[..., :-1, :].contiguous() # NOTE: we don't need the logits of the last token
```
Comment (hlnchen) [inline torchprime/torch_xla_models/flex/modeling_qwen.py:424]: we shift logits in inference_utils at inference time
